coordinator/core/marbleapi_test.go

```diff
@@ -240,18 +286,29 @@ func TestParseSecrets(t *testing.T) {
 	require.NoError(err)
 	assert.EqualValues("AAECAwQFBgcICQoLDA0ODw==", parsedSecret)
 
+	// Check if we can decode a certificate from PEM
+	parsedSecret, err = parseSecrets("{{ pem .Secrets.testcertificate.Cert }}", testWrappedSecrets)
```

Comment:
also add a test for `{{ raw .Secrets.testcertificate.Cert }}` (and implement the feature)